docs/cli.md

```diff
@@ -208,6 +210,56 @@ Run data diagnosis and output the results in jsonl format:
 sb result diagnosis --data-file outputs/results-summary.jsonl --rule-file rule.yaml --baseline-file baseline.json --output-file-format json
 ```
 
+Run data diagnosis and output the results in markdown format with 2 valid decimal places:
+```bash title="SB CLI"
+sb result diagnosis --data-file outputs/results-summary.jsonl --rule-file rule.yaml --baseline-file baseline.json --output-file-format md --round 2
+```
+
+### `sb result summary`
+
+Generate the readable summary report automatically from benchmarking results according to rules defined in rule file.
+
+```bash title="SB CLI"
+sb result summary [--data-file]
+       [--rule-file]
+       [--output-dir]
+       [--output-file-format]
+       [--round]
+```
+
+#### Required arguments
+
+| Name                   | Description            |
+|------------------------|------------------------|
+| `--data-file` `-d`     | Path to raw data file. |
+| `--rule-file` `-r`     | Path to rule file.     |
+
+#### Optional arguments
+
+| Name                   | Default | Description                                                                 |
+|------------------------|---------|-----------------------------------------------------------------------------|
+| `--output-dir`         | `None`  | Path to output directory, outputs/{datetime} will be used if not specified. |
+| `--output-file-format` | `md` | Format of output file, 'excel', 'md' or 'html'. Default: md.                       |
+| `--round` | 2 | Number of valid decimal places to show in output. Default: 2.                       |
```

Comment:
aligment as well. `Number of valid decimal places to show in output. Default: 2. `, what's the meaning of `valid decimal places`?

Reply:
ok, updated